src/services/git/GitHubService.ts

```diff
@@ -71,11 +71,12 @@ export class GitHubService implements ICVSService {
   async getPullRequests(
     owner: string,
     repo: string,
-    options?: ListGetterOptions<{ state?: GitHubPullRequestState }>,
+    options?: ListGetterOptions<{ state?: PullRequestState }>,
   ): Promise<Paginated<PullRequest>> {
     let url = 'GET /repos/:owner/:repo/pulls';
     if (options !== undefined && options.filter !== undefined && options.filter.state !== undefined) {
-      url = `${url}?state=${options.filter.state}`;
+      const state = CSVServicesUtils.getPRState(options.filter.state, CSVService.github);
+      url = `${url}?state=${state}`;
```

Comment:
Do not build URL as a string... It'll be a problem when we have 2 or more query parameters. Use `qs` library - https://www.npmjs.com/package/qs#stringifying